benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -246,13 +249,16 @@ def _load_resource(self, key: str, force: bool = False) -> None:
 
     def evaluate(
         self,
+        run_name: str,
         candidate_generation: bool = True,
         baseline: bool = True,
         context: bool = True,
         spacyfishing: bool = True,
         n_items: Optional[int] = None,
     ) -> None:
         """Evaluates trained pipeline on test set.
+        run_name (str): Run name.
+        highlight_criterion (str): Criterion to highlight in table. One of ("f1", "recall", "precision").
```

Comment:
`highlight_criterion` isn't actually an input parameter of `evaluate`?

Reply:
C-P mistake, removed in next commit.